python-checks/src/main/java/org/sonar/python/checks/CheckUtils.java

```diff
@@ -86,6 +86,8 @@ public static boolean classHasInheritance(AstNode classDef) {
   }
 
   public static boolean isAssignmentExpression(AstNode expression) {
+    // TODO PythonGrammar.ANNASSIGN
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.python%3Apython&branch=SONARPY-247&issues=AWDpbCVsGaMZFZs6hkso&open=AWDpbCVsGaMZFZs6hkso)

[ISSUE_KEY]: AWDpbCVsGaMZFZs6hkso